packages/eslint-plugin-x/README.md

```diff
@@ -6,7 +6,6 @@ custom plugin, but it includes also style formatter and good practices outside E
 
 - [ESLint](https://eslint.org/) and third-party plugins
 - [Prettier](https://prettier.io/)
-- [standard-version](https://github.com/conventional-changelog/standard-version)
```

Comment:
Why are we removing the `standard-version` references from this readme? I've noticed we are not using it anymore in the `monorepo`, but it is still exported and may be used by anyone using this `package`. Should we remove this config entirely from the package?